pandas/tests/reductions/test_reductions.py

```diff
@@ -1035,6 +1035,31 @@ def test_minmax_nat_dataframe(self, nat_df):
         assert nat_df.min(skipna=False)[0] is pd.NaT
         assert nat_df.max(skipna=False)[0] is pd.NaT
 
+    def test_min_max_dt64_with_NaT(self):
+        # Calling the following sum functions returned an error for dataframes but
+        # returned NaT for series. These tests check that the API is consistent in
+        # min/max calls on empty Series/DataFrames. See GH:33704 for more
+        # information
+
+        df = pd.DataFrame(dict(x=pd.to_datetime([])))
+        expected_dt_series = pd.Series(pd.to_datetime([]))
+        # check axis 0
+        assert (df.min(axis=0).x is NaT) == (expected_dt_series.min() is NaT)
+        assert (df.max(axis=0).x is NaT) == (expected_dt_series.max() is NaT)
+
+        # check axis 1
+        tm.assert_series_equal(df.min(axis=1), expected_dt_series)
```

Comment:
I just want to mention that calling `min` on an empty dataframe does not return the same, since this it currently not possible (and I don't think it is desired) with the implementation of reduce-operations. This test checks that when reducing over the index a Dataframe behaves like a series.